datasets/us/cuba_sanctions/crawler.py

```diff
@@ -5,10 +5,13 @@
 from zavod import helpers as h
 from zavod.shed.zyte_api import fetch_html
 
-ORIGINAL_ACCOMMODATIONS_URL = (
-    "https://www.state.gov/cuba-sanctions/cuba-prohibited-accommodations-list/"
-)
+ORIGINAL_ACCOMMODATIONS_URL = "https://www.state.gov/cuba-sanctions/cuba-prohibited-accommodations-list/cuba-prohibited-accommodations-list-initial-publication/"
```

Comment:
Why are we changing this to a broken link? Was this working yesterday? Perhaps the most sensible link for this is https://www.state.gov/cuba-prohibited-accommodations-list-initial-publication/ so that the hash check tells us if they modify the content and we need to update our spreadsheet.
![image](https://github.com/user-attachments/assets/fbab84cf-2e74-4cb3-8934-0decea7ac5a4)